### PR TITLE
Supermatter Cider

### PR DESCRIPTION
**Pull request by ghost**
## Changelog
:cl:
add: Supermatter cider
/:cl:

## About The Pull Request
adds a new alcohol that can only be obtained by adminbus or botany rng.
## Why It's Good For The Game
Have you ever thought about drinking liquid supermatter? this adds that in and it's surprisingly alcoholic, ~~however you must go and find a certain abandoned coffee machine in space as that is the only thing capable of safely dispensing it.~~

### CODE REVIEW

**Comment by ghost** [comment]
The abandoned coffee machine no longer exists.

**Comment by ghost** [comment]
> The abandoned coffee machine no longer exists.

fixed

**Comment by ghost** [comment]
@lordsaladin ???

**Comment by ghost** [comment]
> @lordsaladin ???

changed the description

**Comment by ghost** [comment]
Still no way to obtain this

**Comment by ghost** [comment]
Oh wait

**Comment by ghost** [comment]
should i add a way?

File: hippiestation/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -210,3 +210,29 @@
...
+				M.adjustBrainLoss(5)
+				if(prob(15))
+					M.vomit(20)
+			if(20 to INFINITY)
+				M.adjustBrainLoss(5)
+				if(prob(30))
+					M.vomit(20, 0, 8)
+					if(prob(10))
+						M.spew_organ()
+						dust_mob(user, cause = "drinking fermented supermater")
```
Comment (yoyobatty) [inline hippiestation/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: replace `user` with `M`

File: hippiestation/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 2]
```diff
@@ -210,3 +210,29 @@
 			M.adjustOxyLoss(-0.25, 0)
 			. = 1
 		..()
+		/datum/reagent/consumable/ethanol/SMcider
```
Comment (yoyobatty) [inline hippiestation/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: remove this tab

File: hippiestation/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 3]
```diff
@@ -210,3 +210,29 @@
...
+	name = "Supermatter cider"
+	description = "who would have thought supermatter was alcoholic "
```
Comment (yoyobatty) [inline hippiestation/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: space before "

**Comment by ghost** [comment]
@yoyobatty  fixed your issues

**Comment by WarRage333** [comment]
Botany strange seed buff

image

**Comment by ghost** [comment]
> Botany strange seed buff
> 
> image
i see this as an absolute win

**Comment by mikolaj007** [comment]
+1, i always enjoy op bullshit for botany
actually, yea. if you wanna make this bus-only, you gotta do "can_synth = FALSE"

**Comment by ghost** [comment]
botany buffs are best

**Comment by Manly-Dwarf** [comment]
genius

**Comment by Toxici11i** [comment]
consider this - botany rolls this in strange seed, puts into syringe and fires at people

hue new instakill syringes

**Comment by Manly-Dwarf** [comment]
> Botany strange seed buff
> 
> image

File: hippiestation/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 4]
```diff
@@ -210,3 +210,29 @@
...
+				M.adjustBrainLoss(5)
+				if(prob(15))
+					M.vomit(20)
+			if(20 to INFINITY)
+				M.adjustBrainLoss(5)
+				if(prob(30))
+					M.vomit(20, 0, 8)
+					if(prob(10))
+						M.spew_organ()
+						dust_mob(M, cause = "drinking fermented supermater")
```
Comment (yoyobatty) [inline hippiestation/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: This is actually supposed to be `M.dust()` and put a line below for a seperate log. dust_mub is a proc for the supermatter

**Comment by diegoxt45** [comment]
+1 i rike it

**Comment by yoyobatty** [comment]
Why did you close?

**Comment by sagattyy** [comment]
retards always closing useful stuff

**Comment by ghost** [comment]
@yoyobatty @sagattyy  the user deleted their account.

**Pull request closed without merging**